### ISSUE TITLE
Cannot Alter Default Pools

### ISSUE THREAD

**Issue by NotKoalafied** [BobbyGR/AIOMiner#121]
I want to delete some of the default pools because the miner will periodically try to use them, but deleting them doesn't work. Please add either this feature or the ability to ban the use of certain pools.

**Comment by thepowerfulwoz** [comment]
go to the pool and coin settings, and check the primary pool box

**Comment by NotKoalafied** [comment]
I tried that before making the issue.

**Comment by BobbyGR** [comment]
Default pools can not be deleted unless you delete them from the default json.  Can you give me repeatable steps of how the primary button is not working?

**Comment by NotKoalafied** [comment]
I did a little bit more digging and discovered that I misdiagnosed the issue, so I will be closing it. But as a side note, deleting the default pools from the json doesn't work either as it just rewrites the json on init.

My misdiagnosis stemmed from seeing the standard devfee transactions and thinking that AIMOiner was attempting to shift me to a different pool as the devfee pool is one of the default pools.
